src/event_form/components/calendar/form-event-calendar.component.html

```diff
@@ -14,22 +14,37 @@
 <div class="calendar-timing-info">
     <div class="panel panel-default">
         <div class="panel-body">
-            <div class="calendar-time-spans" ng-if="!calendar.weeklyRecurring">
+            <div class="calender-time-init" ng-show="calendar.showBootstrapTimeSpans && calendar.type === 'single'">
+                <div class="form-group">
+                  <label for="time-span-init-start-date">Wanneer start je evenement?</label>
+                 </div>
+                <div class="form-inline">
+                 <div class="date">
+                   <udb-form-calendar-datepicker ng-model="calendar.bootstrapDate" ng-change="calendar.delayedTimeSpanChanged()">
```

Comment:
`ng-change` is volgens mij overbodig omdat je toch moet bevestigen met een klik op een knop. Het gedrag voor validatie en bewaren kan volledig in de `calendar.confirmBootstrap()` methode?